src/orchestration.d.ts

```diff
@@ -378,8 +395,20 @@ export interface OrchestrationAccount {
    * @returns void
    */
   transferSteps: (amount: AmountArg, msg: TransferMsg) => Promise<void>;
+  /**
+   * deposit payment from zoe to the account. For remote accounts,
+   * an IBC Transfer will be executed to transfer funds there.
+   */
+  deposit: (payment: Payment) => Promise<void>;
 }
 
+type ChainSpecificMethods = {
+  [C in keyof KnownChains]: KnownChains[C]['methods'];
+};
+
+export type OrchestrationAccount<C extends keyof KnownChains> =
+  BaseOrchestrationAccount & ChainSpecificMethods[C];
```

Comment:
the `ChainSpecificMethods` type isn't doing much.

```suggestion
export type OrchestrationAccount<C extends keyof KnownChains> =
  BaseOrchestrationAccount & KnownChains[C]['methods'];
```